server/liveblog/blogs/blogs.py

```diff
@@ -15,8 +15,9 @@
     'guid': metadata_schema['guid'],
     'title': metadata_schema['headline'],
     'description': metadata_schema['description'],
-    'language': Resource.rel('languages', True),
-    'theme': Resource.rel('themes', True),
+    'theme': {
```

Comment:
in `blog_preferences`, theme is defined with a `string` (should be an `ObectId` maybe), but here `theme` is a snapshot of the theme, we copy all the theme definition to prevent any unwanted changes on this blog.